File: JS-Basics/05-Conditional-Statements-Advanced-Lab/p05_Small_Shop.js
function smallShop(input) {
    let articul = input[0];
    let city = input[1];
    let quantity = Number(input[2]);
    let bill = 0;

    switch (city) {
        case 'Sofia':
            if (articul == 'coffee') {
                bill = quantity * 0.50;
            } else if (articul == 'water') {
                bill = quantity * 0.80;

            } else if (articul === 'beer') {
                bill = quantity * 1.20;

            } else if (articul == 'sweets') {
                bill = quantity * 1.45;

            } else if (articul == 'peanuts') {
                bill = quantity * 1.60;

            } else {};
            break;
        case 'Plovdiv':
            if (articul == 'coffee') {
                bill = quantity * 0.40;
            } else if (articul == 'water') {
                bill = quantity * 0.70;

            } else if (articul === 'beer') {
                bill = quantity * 1.15;

            } else if (articul == 'sweets') {
                bill = quantity * 1.30;

            } else if (articul == 'peanuts') {
                bill = quantity * 1.50;

            } else {};
            break;
        case 'Varna':
            if (articul == 'coffee') {
                bill = quantity * 0.45;
            } else if (articul == 'water') {
                bill = quantity * 0.70;

            } else if (articul === 'beer') {
                bill = quantity * 1.10;

            } else if (articul == 'sweets') {
                bill = quantity * 1.35;

            } else if (articul == 'peanuts') {
                bill = quantity * 1.55;

            } else {};
            break;
    }
    console.log(bill);
}

smallShop(["coffee", "Varna", "2"]);
smallShop(["peanuts", "Plovdiv", "1"]);
smallShop(["beer", "Sofia", "6"]);
smallShop(["water", "Plovdiv", "3"]);
smallShop(["sweets", "Sofia", "2.23"]);

File: Programming-Basic/05-Conditional-Statements-Advanced-Lab/p05_Small_Shop.java
import java.util.Scanner;

public class p05_Small_Shop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String product = scanner.nextLine();
        String city = scanner.nextLine();
        double quantity = Double.parseDouble(scanner.nextLine());

        //град / продукт	coffee	water	beer	sweets	peanuts
        //Sofia         	0.50	0.80	1.20	1.45	1.60

        double price = 0.0;

        switch (city) {
            case "Sofia":
                if (product.equals("coffee")) {
                    price = 0.50;
                } else if (product.equals("water")) {
                    price = 0.80;
                } else if (product.equals("beer")) {
                    price = 1.20;
                } else if (product.equals("sweets")) {
                    price = 1.45;
                } else if (product.equals("peanuts")) {
                    price = 1.60;
                }
                break;

            //Plovdiv	        0.40	0.70	1.15	1.30	1.50
            case "Plovdiv":
                if (product.equals("coffee")) {
                    price = 0.40;
                } else if (product.equals("water")) {
                    price = 0.70;
                } else if (product.equals("beer")) {
                    price = 1.15;
                } else if (product.equals("sweets")) {
                    price = 1.30;
                } else if (product.equals("peanuts")) {
                    price = 1.50;
                }
                break;

            //Varna	            0.45	0.70	1.10	1.35	1.55
            case "Varna":
                if (product.equals("coffee")) {
                    price = 0.45;
                } else if (product.equals("water")) {
                    price = 0.70;
                } else if (product.equals("beer")) {
                    price = 1.10;
                } else if (product.equals("sweets")) {
                    price = 1.35;
                } else if (product.equals("peanuts")) {
                    price = 1.55;
                }
                break;
        }
        double finalPrice = quantity * price;
        System.out.println(finalPrice);

    }
}
